README.md

```diff
@@ -5,23 +5,291 @@
++   `verbose`: display entire manifest contents and other debug info (default is false).
+
+#### Basic default manifest
+
+The following command generates a manifest with the compiled in default profile and charts:
+
+```bash
+iop manifest
+```
+
+You can see these sources for the compiled in profiles in the repo under data/profiles, while the compiled in helm
+charts are under data/charts.
+
+#### Output to dirs
+
+The output of the manifest is concatenated into a single file. To generate a directory hierarchy with subdirectory
+levels representing a child dependency, use the following command:
+
+```bash
+iop manifest -o istio_manifests
+```
+
+Use depth first search to traverse the created directory hierarchy when applying your YAML files. Child manifest
+directories must wait for the parent, but not sibling manifest directories.
+
+#### Just apply it for me
+
+The following command generates the manifests and applies them in the correct dependency order, waiting for the
+dependencies to have the needed CRDs available: 
+
+```bash
+iop install
+```
+
+Note: right now the configuration that would be applied is only displayed, since `dry-run` is true by default. Set
+`--dry-run=false` to actually apply the generated configuration to the cluster.
+
+#### Review the values of the current configuration profile
+
+The following command shows the values of the current configuration profile:
+
+```bash
+iop dump-profile
+```
+
+#### Select a specific configuration profile
+
+The simplest customization is to select a profile different to `default` e.g. `sds`. Create the following config file: 
+
+```yaml
+# sds-install.yaml
+
+profile: sds
+```
+
+Use the Istio operator `iop` binary to apply the new configuration profile:
+
+```bash
+iop manifest -f sds-install.yaml
+```
+
+After running the command, the Helm charts are rendered using `data/profiles/sds.yaml`. 
+
+#### Install from file path
+
+The compiled in charts and profiles are used by default, but a file path can be specified, e.g.
+
+```yaml
+profile: file:///usr/home/bob/go/src/github.com/ostromart/istio-installer/data/profiles/default.yaml
+customPackagePath: file:///usr/home/bob/go/src/github.com/ostromart/istio-installer/data/charts/
+```
+
+These can be mixed and matched e.g. use a compiled in profile with local filesystem charts. 
+
+#### New API customization
+
+The [new platform level installation API](https://github.com/istio/operator/blob/95e89c4fb838b0b374f70d7c5814329e25a64819/pkg/apis/istio/v1alpha1/istioinstaller_types.proto#L25) 
+defines install time parameters like feature/component enablement and namespace, and k8s settings like resources, HPA spec etc. in a structured way.   
+The simplest customization is to turn features and components on and off. For example, to turn off all policy:
+
+```yaml
+profile: sds
+policy:
+  enabled: false
+```
+
+Note that unlike helm, all configurations are validated against a schema, so the operator detects syntax errors. Another
```

Comment:
capitalize Helm.

Reply:
I'd reword this sentence as follows "The operator validates the configuration and automatically detects syntax errors.  Helm lacks this capability.  If you are using Helm values that are incompatible, the schema validation used in the operator may likely reject input that is valid for Helm."

Reply:
Capitalized throughout. 
Re: the rewording, that's a great suggestion. Thanks.